doc/mgr/orchestrator.rst

```diff
@@ -411,6 +411,138 @@ Service Commands::
 
     ceph orch <start|stop|restart|redeploy|reconfig> <service_name>
 
+
+.. _orchestrator-haproxy-service-spec:
+
+High availability service for RGW
+=================================
+
+This service allows the user to create a high avalilability RGW service
+providing a mimimun set of configuration options.
+
+The orchestrator will deploy and configure automatically several HAProxy and
+Keepalived containers to assure the continuity of the RGW service while the
+Ceph cluster will have at least 1 RGW daemon running.
+
+The next image explains graphically how this service works:
+
+.. image:: ../images/HAProxy_for_RGW.svg
+
+There are N hosts where the HA RGW service is deployed. This means that we have
+an HAProxy and a keeplived daemon running in each of this hosts.
+Keepalived is used to provide a "virtual IP" binded to the hosts. All RGW
+clients use this  "virtual IP"  to connect with the RGW Service.
+
+Each keeplived daemon is checking each few seconds what is the status of the
+HAProxy daemon running in the same host. Also it is aware that the "master" keepalived
```

Comment:
```suggestion
HAProxy daemon running in the same host. Also it is aware that the "primary" keepalived
```

Reply:
I have used the word "master" because it is also the word used to configure the "primary" keepalived server. I would prefer not to change the "keepalived" nomenclature if this is possible.
Do you think that it is needed any clarification in this point?

Reply:
Good question. I would assume they will move to use the better terminology too. Can you check with the keepalived community?

Reply:
I do not believe that they will be too much receptive to change the terminology used until now for the last 11 years. I think that is better for us just to adapt to the keepalived terminology and concept in our particular use case for keepalived.
If it is not well understood the term "master" in this documentation, we can clarify this,  but i think that is not good to replace the word "master"  for "primary".